native/cocos/2d/renderer/Batcher2d.h

```diff
@@ -0,0 +1,115 @@
+#pragma once
+#include <2d/renderer/UIMeshBuffer.h>
+#include <cocos/2d/renderer/RenderDrawInfo.h>
+#include <cocos/2d/renderer/UIMeshBuffer.h>
+#include <cocos/base/TypeDef.h>
+#include <cocos/core/assets/Material.h>
+#include <cocos/renderer/gfx-base/GFXTexture.h>
+#include <cocos/renderer/gfx-base/states/GFXSampler.h>
+#include <cocos/scene/DrawBatch2D.h>
+#include <unordered_map>
+#include <vector>
```

Comment:
Do not have to include `<vector>` and `<unordered_map>`. Use `ccstd` version instead.

Reply:
done